charts/prefect-server/templates/background-services-deployment.yaml

```
@@ -0,0 +1,156 @@
+      {{- if .Values.backgroundServices.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.backgroundServices.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: background-services
+        prefect-version: {{ .Chart.AppVersion }}
+        {{- if .Values.backgroundServices.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.backgroundServices.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- if .Values.global.prefect.image.pullSecrets }}
+      imagePullSecrets:
+      {{- range .Values.global.prefect.image.pullSecrets }}
+        - name: {{ . }}
+      {{- end }}
+      {{- end }}
+      serviceAccountName: {{ template "backgroundServices.serviceAccountName" . }}
+      {{- if .Values.backgroundServices.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.backgroundServices.affinity "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.backgroundServices.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.backgroundServices.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.backgroundServices.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.backgroundServices.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.backgroundServices.podSecurityContext }}
+      securityContext: {{- .Values.backgroundServices.podSecurityContext | toYaml | nindent 8 }}
+      {{- end }}
+      {{- if .Values.backgroundServices.priorityClassName }}
+      priorityClassName: {{ .Values.backgroundServices.priorityClassName }}
+      {{- end }}
+      containers:
+        - name: prefect-background-services
+          image: "{{ .Values.global.prefect.image.repository }}:{{ .Values.global.prefect.image.prefectTag }}"
+          imagePullPolicy: {{ .Values.global.prefect.image.pullPolicy }}
+          command:
+            - /usr/bin/tini
+            - -g
+            - --
+            - /opt/prefect/entrypoint.sh
+          args:
+            - prefect
+            - server
+            - services
+            - start
+          workingDir: /home/prefect
+          env:
+            - name: HOME
+              value: /home/prefect
+            - name: PREFECT_API_URL
+              value: {{ .Values.global.prefect.prefectApiUrl | quote }}
+            - name: PREFECT_DEBUG_MODE
+              value: {{ .Values.backgroundServices.debug | quote }}
+            - name: PREFECT_LOGGING_SERVER_LEVEL
+              value: {{ .Values.backgroundServices.loggingLevel | quote }}
+            - name: PREFECT_SERVER_API_HOST
+              value: {{ .Values.global.prefect.prefectApiHost | quote }}
+            - name: PREFECT_UI_ENABLED
+              value: false
```

```suggestion
              value: 'false'
```
Resolves: `Error: INSTALLATION FAILED: 1 error occurred:
        * Deployment in version "v1" cannot be handled as a Deployment: json: cannot unmarshal bool into Go struct field EnvVar.spec.template.spec.containers.env.value of type string`